### PR TITLE
[14.0][MIG] currency_rate_update_xe

### PR DESCRIPTION
**Pull request by mpascuall**
Module migrated to version 14.0

cc https://github.com/APSL 153469

@miquelalzanillas @lbarry-apsl @javierobcn @peluko00 please review

### CODE REVIEW

**Review by javierobcn — APPROVED** [review]
Functional Ok, checked in runboat

**Comment by miquelalzanillas** [comment]
Functionallly working!

**Review by miquelalzanillas — APPROVED** [review]

**Review by peluko00 — APPROVED** [review]

**Review by lbarry-apsl — APPROVED** [review]
Works well on the Runboat

File: currency_rate_update_xe/tests/test_currency_rate_update_xe.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Copyright 2023 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import fields
+from odoo.tests import common
+
+
+class TestResCurrencyRateProviderXE(common.TransactionCase):
+    def setUp(self):
```
Comment (pedrobaeza) [inline currency_rate_update_xe/tests/test_currency_rate_update_xe.py]: You shouldn't change from `setUpClass` to `setUp`, as it's less efficient.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please put the migration commit message properly with the module name, that is missing.

File: currency_rate_update_xe/tests/test_currency_rate_update_xe.py (hunk 1 above)
Comment (miquelpascual) [reply currency_rate_update_xe/tests/test_currency_rate_update_xe.py]: Fixed please review

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**